internal/converter/openapi_reconciler.go

```diff
@@ -0,0 +1,56 @@
+package converter
+
+import "github.com/kubeflow/model-registry/pkg/openapi"
+
+// NOTE: methods must follow these patterns, otherwise tests could not find possible issues:
+// Converters patch fields entity: UpdateExisting<ENTITY>
+
+// goverter:converter
+// goverter:output:file ./generated/openapi_reconciler.gen.go
+// goverter:wrapErrors
+// goverter:enum:unknown @error
+// goverter:matchIgnoreCase
+// goverter:useZeroValueOnPointerInconsistency
+type OpenAPIReconciler interface {
```

Comment:
this "reconciler" could have also been named "updater" and the scope is given an existing (from MLMD db) entity, applies the "patch" as in the valorized fields values, returning the "to-be" state which is what the Core layer but most-importantly MLMD requires 👍

Reply:
As soon as we have [copy constructors](https://github.com/jmattheis/goverter/issues/147) I intend to simplify this piece of code again, so we can think of something together.